src/game/Editor/Editor_Taskbar_Utils.cc

```diff
@@ -209,7 +209,8 @@ static void ShowEditorToolbar(TaskMode const iNewTaskMode)
 		case TASK_MERCS:			ShowEditorButtons( FIRST_MERCS_TEAMMODE_BUTTON, LAST_MERCS_TEAMMODE_BUTTON );	break;
 		case TASK_MAPINFO:		ShowEditorButtons( FIRST_MAPINFO_BUTTON, LAST_MAPINFO_BUTTON );								break;
 		case TASK_OPTIONS:		ShowEditorButtons( FIRST_OPTIONS_BUTTON, LAST_OPTIONS_BUTTON );								break;
-		default:		return;
+		default:
+			return;
```

Comment:
the cases are not aligned here, and maybe we should split the 'breaks' to the next line